tidb-cloud/tidb-cloud-quickstart.md

```diff
@@ -44,11 +44,37 @@ In Chat2Query, you can either simply type `--` followed by your instructions to
 
 3. In the editor, you can either simply type `--` followed by your instructions to let AI automatically generate SQL queries, or write SQL queries manually.
 
+   Example:
+   If you would like to create a new table test.t with two columns (Column id and Column name), you can just press the Enter key at the end of your instructions. Then the AI will generate a SQL. Press Tab key to accept the AI-generated SQL, or any other key to cancel. Then you can run the SQL.
+
```

Comment:
Can we move the example code in Line 51~Line 71 after step 5? Because users might need to know how to run SQL first. 

After the `CREATE TABLE` statement is created by AI, if users forget to run it and directly type `-- add 3 rows`, there will be an error reporting no table is available.

Reply:
Great suggestion, I will follow it.

Reply:
@Jellybeanvino  On second thought, how about placing the full example after the detailed instructions for running a query?


![image](https://github.com/pingcap/docs/assets/79440533/c687caa1-afed-4bdf-a085-d64c313875fa)

Reply:
There is already an example placed after step 4. I think placing one more example here would be tedious and needless.

Reply:
@Jellybeanvino, sorry for the confusion, I mean moving the full example in L57~77 after L105

Reply:
Done